exporter/exporterhelper/logs.go

```diff
@@ -55,6 +68,10 @@ func (req *logsRequest) export(ctx context.Context) error {
 	return req.pusher(ctx, req.ld)
 }
 
+func (req *logsRequest) marshall() ([]byte, error) {
```

Comment:
```suggestion
func (req *logsRequest) marshal() ([]byte, error) {
```

and accordingly elsewhere